customer-service/omnichannel/real-time-translation/webResourceV2.js

```diff
@@ -9,464 +9,695 @@ If you use the sample script and enable data collection in your applications, yo
  
 You can learn about Microsoft’s own data collection and use in the applicable product documentation and the Microsoft Privacy Statement at http://go.microsoft.com/fwlink/?LinkID=521839.  You agree to comply with all applicable provisions of the Microsoft Privacy Statement.
 
+ */
+const oneSecondInMs = 1000;
+const oneMinuteInMs = 60 * oneSecondInMs;
+const oneHourInMs = 60 * oneMinuteInMs;
+const oneSecondInTicks = 10000000;
+var Sdk = window.top.Sdk || {};
```

Comment:
Is the name "window.top.Sdk" used anywhere else? Just thinking maybe we need to have a better/more specific name.

Reply:
We are using window.top elsewhere, I have left this as is, let me know if you think it's better to use as a constant